### PR TITLE
Add script for cherry-pick

### PR DESCRIPTION
**Pull request by fisherxu**
Add script for cherry-pick, to cherry pick docs to different versions.

ref: copy from kubeedge/kubeedge/hack

### CODE REVIEW

File: hack/cherry_pick_pull.sh [hunk 1]
```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env bash
+
+# Copyright 2015 The Kubernetes Authors.
```
Comment (Shelley-BaoYue) [inline hack/cherry_pick_pull.sh:3]: Do we need change the date to 2024?

Comment (fisherxu) [reply hack/cherry_pick_pull.sh:3]: Probably not, it's a full copy from K8s, ref: 
https://github.com/kubeedge/kubeedge/blob/master/hack/cherry_pick_pull.sh

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

**Review by wbc6080 — COMMENTED** [review]
/lgtm

**Review by Shelley-BaoYue — COMMENTED** [review]
/approve

**Pull request merged**